sql/sqlserver/server.go

```diff
@@ -674,22 +676,25 @@ func (s *Server) processColumns(desc *structured.TableDescriptor,
 
 func (s *Server) processInsertRows(node parser.SelectStatement) (rows []sqlwire.Result_Row, err error) {
 	switch nt := node.(type) {
+	// case *parser.Select:
+	// case *parser.Union:
+	// TODO(vivek): return s.query(nt.stmt, nil)
 	case parser.Values:
 		for _, row := range nt {
 			var vals []sqlwire.Datum
 			for _, val := range row {
 				switch vt := val.(type) {
 				case parser.StrVal:
-					tmp := string(vt)
-					vals = append(vals, sqlwire.Datum{StringVal: &tmp})
-				case parser.NumVal:
-					tmp := string(vt)
-					vals = append(vals, sqlwire.Datum{StringVal: &tmp})
-				case parser.ValArg:
-					return rows, util.Errorf("TODO(pmattis): unsupported node: %T", val)
+					vals = append(vals, sqlwire.Datum{StringVal: (*string)(&vt)})
```

Comment:
I just went ahead and did that. Can you PTAL, especially `EvalExpr` turning a single-element tuple into its single element? That one strikes me as odd. I think I commented about it before, but can't seem to find your reply now.